src/main/java/seedu/address/logic/Messages.java

```diff
@@ -45,6 +45,14 @@ public static String format(Person person) {
                 .append(person.getAddress())
                 .append("; Tags: ");
         person.getTags().forEach(builder::append);
+
+        // append notes if available
+        if (!person.getNotes().getNotes().isEmpty()) {
+            builder.append("; Notes: ");
+            String notesString = String.join(", ", person.getNotes().getNotes());
```

Comment:
Would it be possible to restructure so that that it do not access data through a middle man?

Reply:
I can change notes accessor attribute to public so it'll look like this `person.getNotes().notes` instead. Is this better?

Reply:
That's not the problem, my main concern is the principal "Law of Demeter" (details in week 9 topics). Which states that objects should only interact with objects adjacent to them. In this case, `Message` should only be interacting with `Person` not with `Notes`.

One possible solution is for the `Person.getNotes()` to return a `set<string>` similar to how tag is done.